Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```
@@ -18,22 +18,30 @@ def __init__(self, id, key, secret, url):
         self.id = id
```

we haven't changed `AS_api` so I can't comment directly on the line, but on line 58 we have a print statement

```
print(response.text)
```

can we remove that please?

Also in the lines below that - if we don't get a success status code we should probably throw an exception. You can use `response.raise_for_status()` to do this automatically for you if you want.

This would avoid you from marking a batch as successful and incrementing the `event_num` - which is what will currently happen as we aren't throwing an exception if the request to Azure fails.